src/docs/get-started/codelab-web.md

```diff
@@ -2,6 +2,9 @@
 title: Write your first Flutter app on the web
```

Comment:
Towards the end of the doc there's a visible "[todo: replace screenshot]". Do you think you could do that? Please?  :)

Reply:
fixed, I updated the other DevTools images to use an empty chrome shell